### DISCUSSION TITLE
How to set up the proxy server?

### DISCUSSION THREAD

**Discussion by zhangxianwei2015** [acheong08/ChatGPT#786] [General]
Hi,

Can someone help explain why this proxy needs to be set?
And how to set up this proxy?

PROXY_URL = os.environ.get("PROXY_URL") or "https://chat.duti.tech"

Thanks!

**Comment by lozn00** [comment]
i need binrary file  to run ,can you provider exe file

**Comment by acheong08** [comment]
Cannot distribute
